numpy/ma/tests/test_core.py

```diff
@@ -1868,16 +1868,25 @@ def test_extremum_fill_value(self):
         assert_equal(test['B']['BB'], default_fill_value(a['B']['BB']))
 
         test = minimum_fill_value(a)
-        assert_equal(test[0], minimum_fill_value(a['A']))
-        assert_equal(test[1][0], minimum_fill_value(a['B']['BA']))
-        assert_equal(test[1][1], minimum_fill_value(a['B']['BB']))
-        assert_equal(test[1], minimum_fill_value(a['B']))
+        assert_equal(test['A'], minimum_fill_value(a['A']))
+        assert_equal(test['B']['BA'], minimum_fill_value(a['B']['BA']))
+        assert_equal(test['B']['BB'], minimum_fill_value(a['B']['BB']))
+        assert_equal(test['B'], minimum_fill_value(a['B']))
```

Comment:
If you make the return value of `minimum_fill_value` a scalar instead of a 0d array, this test does not need to change since scalar fields can be indexed with integers. Eg, try replacing `test` here by `test = test[()]`.